### PR TITLE
Fix typos on website's plugin pages, and in corresponding READMEs

### PR DESCRIPTION
**Pull request by 3nk1du**
## Summary
Make website and README changes for plugins:
1. Fix typos in the **SubGraph plugin**, making sure to make the same changes in the corresponding README.
2. Fix typos in the **Mocks plugin**, making sure to make the same changes in the corresponding README.
3. Remove Mocks plugin info from **SubGraph plugin** documentation.

Ignore anything that could change meaning.

### CODE REVIEW

File: website/pages/docs/plugins/sub-graph.mdx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const combinedSchema = builder.toSchema({ subGraph: ['Internal', 'Public'] });
   to being part of the same sub-graph as their parent type. Only applies when type does not have
   `defaultForFields` set.
 
-You can mock any field by adding a mock in the options passed to `builder.buildSchema` under
+You can mock any field by adding a mock in the options passed to `builder.toSchema` under
```
Comment (3nk1du) [inline website/pages/docs/plugins/sub-graph.mdx:80]: I believe you meant to `toSchema` here. Also would you like me to reference the Mocks plugin here, or do you expect this to work out of the box for the SubGraph plugin (I haven't used it yet).

The change would read:

You can mock any field by using the [Mocks plugin](https://pothos-graphql.dev/docs/plugins/mocks) and adding a mock in the options passed to `builder.toSchema` under...

Comment (hayes) [reply website/pages/docs/plugins/sub-graph.mdx:80]: I think this was accidentally copied from the mocks plugin, and should just be removed.  The sub-graph plugin doesn't have any special relationship with mocking, and probably shouldn't mention it in the docs.

**Review by hayes — APPROVED** [review]
Thank you so much for all these fixes, I really appreciate it!  I know these docs were in a pretty rough state

File: website/pages/docs/plugins/sub-graph.mdx (hunk 1 above)
Comment (3nk1du) [reply website/pages/docs/plugins/sub-graph.mdx]: Good, that makes more sense :)

**Comment by 3nk1du** [comment]
> Thank you so much for all these fixes, I really appreciate it! I know these docs were in a pretty rough state

I'm enjoying it. The docs I've read so far have been clear and well written. I don't think I've had to go digging in the code yet for anything, which is surprising. So, compliments to the author!

**Pull request merged**